centreon/tests/e2e/fixtures/dashboards/creation/widgets/dashboadWithDoubleWidget.json

```diff
@@ -0,0 +1,220 @@
+                  "enabled":true
+               },
+               "name":"single metric",
+               "openLinksInNewTab":true
+            }
+         },
+         "widget_type":"/widgets/singlemetric"
+      },
+      {
+         "layout":{
+            "height":3,
+            "min_height":3,
+            "min_width":3,
+            "width":12,
+            "x":0,
+            "y":3
+         },
+         "name":"centreon-widget-singlemetric",
+         "widget_settings":{
+            "data":{
+               "resources":[
+                  {
+                     "resourceType":"service",
+                     "resources":[
+                        {
+                           "uuid":"h14-s26",
+                           "duration":"46m 12s",
+                           "last_check":"1m 12s",
+                           "short_type":"s",
+                           "id":26,
+                           "type":"service",
+                           "name":"Ping",
+                           "alias":null,
+                           "fqdn":null,
+                           "host_id":14,
+                           "service_id":26,
+                           "icon":null,
+                           "monitoring_server_name":"Central",
+                           "parent":{
+                              "uuid":"h14",
+                              "id":14,
+                              "name":"Centreon-Server",
+                              "type":"host",
+                              "short_type":"h",
+                              "status":{
+                                 "code":0,
+                                 "name":"UP",
+                                 "severity_code":5
+                              },
+                              "alias":"Monitoring Server",
+                              "fqdn":"127.0.0.1",
+                              "monitoring_server_name":null
+                           },
+                           "status":{
+                              "code":0,
+                              "name":"OK",
+                              "severity_code":5
+                           },
+                           "is_in_downtime":false,
+                           "is_acknowledged":false,
+                           "has_active_checks_enabled":true,
+                           "has_passive_checks_enabled":false,
+                           "last_status_change":"2023-11-07T09:22:46+01:00",
+                           "tries":"1/3 (H)",
+                           "information":"OK - 127.0.0.1 rta 0.019ms lost 0%",
+                           "performance_data":null,
+                           "is_notification_enabled":false,
+                           "severity":null,
+                           "links":{
+                              "endpoints":{
+                                 "details":"/centreon/api/latest/monitoring/resources/hosts/14/services/26",
```

Comment:
are these numbers (14, 26) consistent? shouldn't you parametrize them one way or another?